h/static/scripts/directive/thread.coffee

```diff
@@ -13,7 +13,8 @@ uuid = require('node-uuid')
 # the collapsing behavior.
 ###
 ThreadController = [
-  ->
+  '$scope',
```

Comment:
Is `// @ngInject` supported in CoffeeScript sources (ie. does that transform run before or after CS processing)?